tests/validators/url_validator_test.py

```diff
@@ -257,3 +257,9 @@ def test_url_with_boolean_options_invalid(require_tld, allow_ip, allow_userinfo,
             'code': 'invalid_url',
             'reason': error_reason,
         }
+
```

Comment:
Can you add a test for the `max_length` parameter too? I think a single test like "if max_length=20, a string of length 21 should raise a string_too_long error" or similar should suffice, since the max_length parameter is tested in detail in the StringValidator already.

Reply:
Okay. I wasn't sure whether to add that test because StringValidator is already thorough in testing that. Will add it.